src/main/java/picard/sam/CreateSequenceDictionary.java

```diff
@@ -146,6 +146,14 @@ public SAMSequenceDictionary makeSequenceDictionary(final File referenceFile) {
         if (URI == null) {
             URI = "file:" + REFERENCE.getAbsolutePath();
         }
+        if (OUTPUT == null) {
+            // determine the name for the dict file in the same way as CachingIndexedFastaSequenceFile.checkAndCreate
```

Comment:
Removed the mention to the GATK class, and added a package-protected method with a TODO to pass to HTSJDK.